sdk/eventhub/ci.data.yml

```diff
@@ -9,7 +9,7 @@ trigger:
     - release/*
   paths:
     include:
-    - sdk/eventhub/ci.data.yml
+    - sdk/eventhub/*.*
```

Comment:
@weshaggard had some strong feelings around not including wildcards.    I'm not sure why we would want to trigger on any root file.    For example, when the migration guide or test resources change.    

I also don't think we need to trigger the T1 pipeline when the CI configuration for T2 changes... but maybe you have a scenario in mind that I'm not thinking o?

Reply:
I don't have a strong opinion on this. But my thought was because we are not great with judging what files actually affect builds, for example `service.projects` wasn't in the initial list, if someone would add Directory.props that would be skipped as well.

Reply:
I'd rather overbuild than let breaks slip in.

Reply:
I don't think this wildcard works.

Reply:
I also don't have a strong opinion;  I'm good either way.   I would suggest that we exclude some of the docs and assorted files explicitly if we go with the include-by-default approach.